app/controllers/api/v1/users/loans_controller.rb

```diff
@@ -11,4 +11,23 @@ def create
       render json: { error: "Unable to create loan" }, status: 422
     end
   end
+
+  def update
+    user = User.find(params[:id])
+    loans = Loan.where(owner_id: user.id).or(Loan.where(borrower_id: user.id))
```

Comment:
this logic is not needed since we have the loan_id being passed through params:
loan = Loan.find(params[:id])